ios/RNTrackPlayer/RNTrackPlayer.swift

```diff
@@ -20,6 +20,9 @@ public class RNTrackPlayer: RCTEventEmitter, AudioSessionControllerDelegate {
     private let audioSessionController = AudioSessionController.shared
     private var shouldEmitProgressEvent: Bool = false
 
+    private var lastForwardJumpInterval: NSNumber = 0;
```

Comment:
@fivecar my only big question here is, are we sure the default of `0` is appropriate here before anything is passed in? It looks like before it would've been set to `nil`? Can you verify that defaulting to `0` is functionally equivalent to how it was behaving before?

Reply:
Great point -- I've now followed your previous suggestion, which is to preserve nil-hood. Thank you!